lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb

```diff
@@ -802,14 +811,20 @@ def create_zone_diffuser(model, zone_diffuser_type, zone)
 
   # =============================================================================================================================
   # create zonal heating equipment
-  def create_zone_htg_eqpt(model, zone_htg_eqpt_type)
+  def create_zone_htg_eqpt(model, zone_htg_eqpt_type, hw_loop)
     always_on = model.alwaysOnDiscreteSchedule
     always_off = model.alwaysOffDiscreteSchedule
     htg_eqpt = nil
     case zone_htg_eqpt_type.downcase
     when 'baseboard_electric'
       htg_eqpt = OpenStudio::Model::ZoneHVACBaseboardConvectiveElectric.new(model)
       htg_eqpt.setName('Zone HVAC Baseboard Convective Electric')
```

Comment:
to be consistent please remove the spaces in the names of the baseboards (elec and hw)